spark/dl/src/main/scala/com/intel/analytics/bigdl/example/modeludf/README.md

```diff
@@ -0,0 +1,516 @@
+## Summary
+ This example is to show how to create a user defined function to do the text classification with BigDL, and use this UDF in Spark SQL (spark 1.5, spark 1.6) and structured streaming (spark 2.0+).
```

Comment:
how to load BigDL model as UDF to perform predictions in Spark SQL / Dataframes and Structured Streaming (Spark 2.0+ only).